OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/ThreadRelatedCallsAnalysis.scala

```diff
@@ -59,39 +59,42 @@ class ThreadStartAnalysis private[cg] (
         isDirect:        Boolean
     ): ProperPropertyComputationResult = {
         val indirectCalls = new IndirectCalls()
-
+        val vmReachableMethods = new VMReachableMethods()
```

Comment:
I think you should be able to just extend the indirect calls above with `with VMReachableMethodsBase, which might or might not be cleaner.`